test/Serilog.ApprovalTests/Serilog.ApprovalTests.csproj

```diff
@@ -0,0 +1,19 @@
+<Project Sdk="Microsoft.NET.Sdk">
+
+  <PropertyGroup>
+    <TargetFramework>net6.0</TargetFramework>
+  </PropertyGroup>
+
+  <ItemGroup>
+    <PackageReference Include="Microsoft.NET.Test.Sdk" Version="17.3.1" />
+    <PackageReference Include="xunit.runner.visualstudio" Version="2.4.3" PrivateAssets="all" />
+    <PackageReference Include="xunit" Version="2.4.2" />
+    <PackageReference Include="Shouldly" Version="4.1.0" />
```

Comment:
I can say nothing about the former, have never used it. I use Shouldly in several OSS projects and it showed itself well.

Reply:
@sungam3r you should use Verify :)

Reply:
Why and how?

Reply:
![изображение](https://user-images.githubusercontent.com/21261007/205699336-03afb6a9-143a-40af-850b-041ceb9d3021.png)
![изображение](https://user-images.githubusercontent.com/21261007/205699397-1d6ec191-5d76-49ec-9cf7-af437989c46e.png)

I will not take any actions until additional information about why Verify should be used instead of Shouldly.

Reply:
i was joking. shouldly is fine. both shouldy and verify rely on diffengine so have many features in common

on many project i use both shouldy  and Verify. since shouldy is better at tradition asserts, and Verify is better at snapshot testing

Reply:
ok

Reply:
@SimonCropp built/maintains Verify :-)

![image](https://user-images.githubusercontent.com/342712/207229019-248ea9c1-0012-4a2c-8206-63f6f2cb1e35.png)